src/human_aware_rl/ppo/ppo_rllib_client.py

```diff
@@ -341,11 +341,17 @@ def my_config():
 
 
 def run(params):
+    layout = params["environment_params"]["mdp_params"]["layout_name"]
+    conf = params["experiment_name"]
+    if params["verbose"]:
+        import wandb
+
+        wandb.init(project=layout, sync_tensorboard=True)
```

Comment:
Wandb project should be the same across all runs probably – this can be a hardcoded global variable imported from utils?

Reply:
Wandb will overlay graphs from the same project on top of each other, so I thought it is probably a good idea to log the experiments based on layouts?

Reply:
I think that's fine – it's very easy to filter out the graphs that one doesn't want to look at or group them as one wishes! We can look through an example together at some point! 😄

Reply:
Got it. I'll remove those.